src/config/index.ts

```diff
@@ -18,6 +22,10 @@ export const STATE_DEPENDENCIES: StateDependencies = {};
 export const STATE_FILE_NAME = "state.json";
 export const STATE_FILE_PATH = `${REPOS_DIRECTORY_PATH}/${STATE_FILE_NAME}`;
 
+export const TEAMS_FILE_NAME = "teams.json";
+export const TEAMS_FILE_PATH = `${REPOS_DIRECTORY_PATH}/${TEAMS_FILE_NAME}`;
+export const TEAMS_DATA: TeamsData = {};
```

Comment:
V minor semantics... move `TEAMS_DATA` to line 25 to maintain consistent format with `STATE_DEPENDENCIES`